### ISSUE TITLE
Create new folder in the currently selected folder

### ISSUE THREAD

**Issue by danielo515** [ozntel/file-tree-alternative#190]
Hello.
This is a great plugin,thank you.
I mostly use this plugin from mobile, and the contextual menu doesn't work.
It thought that by selecting one folder,then tapping the create folder button, the folder will be nested, but instead it goes to the root. Will be cool to fix this, or add another button in the files pane to create a nested folder there.

Thabks

**Comment by jvdevlab** [comment]
I have the same issues on Android. 

- Folders are always created in the root. No matter where you create it from.
- When you press and hold on the file or folder the context menu flicker for a split second and disappears (so none of the operations from it are available)

**Comment by pablo-hp** [comment]
Concordo plenamente com este recurso. É uma lástima ter que criar pastas com o botão direito uma vez que o botão de criar pastas somente cria pastas no diretório raiz e não na pasta selecionada. O plugin realmente precisa de melhorias e uma equipe com conhecimento para tentar ajudar a melhorar ele.

**Comment by jankoweb** [comment]
I have the same problem, reported here - [https://github.com/ozntel/file-tree-alternative/issues/222#issuecomment-2153116103](url)
